pyre/bench/synth/class_attrs_methods.py

```
@@ -25,3 +28,19 @@ def main():
 
+
+class Holder:
+    tag = 13
+
+
+def hot_type_getattr(n):
+    """Hot `getattr(type, name)`, the `builtin_type_getattr` fold."""
+    s = 0
+    i = 0
+    while i < n:
+        s += getattr(Holder, "tag")
```

Ruff (0.16.1)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)